### PR TITLE
[ch915] bigquery: adding support for storing SELECT query results to csv

### PR DESCRIPTION
**Pull request by damjanovl**

### CODE REVIEW

**Comment by blakeburch** [comment]
All downloading functionality seems sound.

1) This one has the same error as uploads, where providing the JSON string results in `Error accessing Google Drive with service account.....`

File: database/bigquery/store_query_results.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import os
+import json
+import tempfile
+import argparse
+
+import pandas as pd
```
Comment (ericelsken) [inline database/bigquery/store_query_results.py:6]: Is this even being used?

File: database/bigquery/store_query_results.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+    return combined_name
+
+
+def create_csv(query, client, destination_file_path):
+    """
+    Read in data from a SQL query. Store the data as a csv.
+    """
+    try:
+        data = client.query(query).to_dataframe()
```
Comment (ericelsken) [inline database/bigquery/store_query_results.py:61]: I'm assuming this is done in an efficient manner? Is there a need to chunk or anything. There sdk will take care of it?

Comment (damjanovl) [reply database/bigquery/store_query_results.py:59]: Neither Google's bigquery default API, or their pandas version of it supports chunked downloads, while it does support chunked uploads but that doesn't help us in this case.

Comment (blakeburch) [reply database/bigquery/store_query_results.py:59]: Confirming that there's no better way to handle this. I built out the "Export to GCS function" to follow the Google preferred method for exporting.

**Pull request merged**